plugins/analytics-module-segment/README.md

```diff
@@ -0,0 +1,210 @@
+    factory: ({ configApi, identityApi }) =>
+      SegmentAnalytics.fromConfig(configApi, {
+        identityApi,
+      }),
+  }),
+];
+```
+
+#### Optional: Configure a user transform
+
+By default, this analytics plugin [identifies][identify] the user taking actions as the logged in Backstage User's entity reference string (e.g. `user:development/guest`). Currently, no other information is provided to the identify call.
+
+To anonymize users, a `userIdTransform` can be provided in one of two ways:
+
+1. The string value `sha-256`
+2. A custom transformation function that matches the contract of `(userEntityRef: string) => Promise<string>`
+
+If `sha-256` is provided, the user entity reference will be pseudonymized into a sha256 string value.
+
+```ts
+// packages/app/src/apis.ts
+import {
+  analyticsApiRef,
+  configApiRef,
+  identityApiRef,
+} from '@backstage/core-plugin-api';
+import { SegmentAnalytics } from '@segment/backstage-plugin-analytics-module-segment';
+
+export const apis: AnyApiFactory[] = [
+  // Instantiate and register the SegmentAnalytics API Implementation.
+  createApiFactory({
+    api: analyticsApiRef,
+    deps: { configApi: configApiRef, identityApi: identityApiRef },
+    factory: ({ configApi, identityApi }) =>
+      SegmentAnalytics.fromConfig(configApi, {
+        identityApi,
+        userIdTransform: 'sha-256',
+      }),
+  }),
+];
+```
+
+For enhanced security, providing a custom transformation function can be used to hash the value in any means desired.
+
+```ts
+// packages/app/src/apis.ts
+import {
+  analyticsApiRef,
+  configApiRef,
+  identityApiRef,
+} from '@backstage/core-plugin-api';
+import { SegmentAnalytics } from '@segment/backstage-plugin-analytics-module-segment';
+
+export const apis: AnyApiFactory[] = [
+  // Instantiate and register the SegmentAnalytics API Implementation.
+  createApiFactory({
+    api: analyticsApiRef,
+    deps: { configApi: configApiRef, identityApi: identityApiRef },
+    factory: ({ configApi, identityApi }) =>
+      SegmentAnalytics.fromConfig(configApi, {
+        identityApi,
+        async userIdTransform(userEntityRef: string) {
+          const salt = configApi.getString(
+            'custom.config.analytics.userIdSalt',
+          );
+          const textToChars = (text: string) =>
+            text.split('').map(c => c.charCodeAt(0));
+          const byteHex = (n: number) =>
+            ('0' + Number(n).toString(16)).substring(-2);
+          const applySaltToChar = (code: number) =>
+            textToChars(salt).reduce((a, b) => a ^ b, code);
+
+          return textToChars(userEntityRef)
+            .map(applySaltToChar)
+            .map(byteHex)
+            .join('');
+        },
+      }),
+  }),
+];
+```
+
+#### Optional: Prevent user identification
+
+If you wish to chose to identify Backstage users in analytics events, simply neglect to provide the `identityApi` when initializing the `SegmentAnalytics` API.
```

Comment:
Phrasing, and possibly reversed sense: "If you choose not to identify..."

Reply:
Ah, yeah, I missed a "not" in here

Reply:
Resolved in https://github.com/segmentio/segment-backstage-plugins/pull/12/commits/d39a9ca380fa3af95719bd9055a22cdf92d317ec